account_product_fiscal_classification/models/wizard_change_fiscal_classification.py

```diff
@@ -27,11 +27,14 @@ def _default_old_fiscal_classification_id(self):
         required=True, domain="[('id', '!=', old_fiscal_classification_id)]")
 
     # View Section
-    @api.one
```

Comment:
In fact, I think api.one is not deprecated anymore, look at all the compute methods in account/models/account_invoice.py for instance, they all use the @api.one formalism. Can we not switch back to @api.one and the former impl assuming self is the record?